rental.rb

```diff
@@ -0,0 +1,12 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
```

Comment:
- [ ] Getters and Setters for @book and @person instance variables should be present as per requirement. Kindly move these from `attr_reader` to `attr_accessor`
e.g
```
`attr_accessor :date, :book, :person`
```